### ISSUE TITLE
[twitter] can no longer download videos

### ISSUE THREAD

**Issue by LeaPhant** [yt-dlp/yt-dlp#7850]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Provide a description that is worded well enough to be understood

Since today yt-dlp reports `No video could be found in this tweet` when used with tweets that do have videos in them.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://twitter.com/ReeS3_Slim2/status/1689756310219247616']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481]
[debug] Python 3.11.3 (CPython x86_64 64bit) - Linux-6.4.5-273-tkg-cfs-x86_64-with-glibc2.38 (OpenSSL 3.1.2 1 Aug 2023, glibc 2.38)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: Cryptodome-3.12.0, certifi-2023.07.22, mutagen-1.46.0, secretstorage-3.3.3, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[twitter] Extracting URL: https://twitter.com/ReeS3_Slim2/status/1689756310219247616
[twitter] 1689756310219247616: Downloading syndication JSON
[twitter] Some metadata is missing without authentication. Use --cookies, --cookies-from-browser, --username and --password, --netrc-cmd, or --netrc (twitter) to provide account credentials
ERROR: [twitter] 1689756310219247616: No video could be found in this tweet
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 710, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/twitter.py", line 1276, in _real_extract
    self.raise_no_formats('No video could be found in this tweet', expected=True)
  File "/usr/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 1188, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)
```

**Comment by bashonly** [comment]
Fixed by 92315c03774cfabb3a921884326beb4b981f786b

If you don't want to wait for next stable release, run `yt-dlp --update-to nightly` or [install the master branch](https://github.com/yt-dlp/yt-dlp/wiki/Installation#pip-master)

Or as workaround with version 2023.07.06 you can [pass cookies](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp)

**Comment by pke** [comment]
nighlty works again. I always got a 404 before.

**Comment by jakeogh** [comment]
On git latest, this is still happening. Below is a clean venv at fcd6a76adc49d5cd8783985c7ce35384b72e545f

```
$ cat ~/_myapps/yt-dlp/install_venv.sh
#!/bin/sh

set -o nounset
set -x

venv="$HOME/.python_venv3.11/yt-dlp"

mkdir -p "${venv}"
python3.11 -m venv "${venv}"
. "${venv}/bin/activate"
#pip install --force-reinstall git+https://github.com/yt-dlp/yt-dlp
pip install git+https://github.com/yt-dlp/yt-dlp

echo ". ${venv}/bin/activate"

$ ~/_myapps/yt-dlp/install_venv.sh
+ venv=/home/user/.python_venv3.11/yt-dlp
+ mkdir -p /home/user/.python_venv3.11/yt-dlp
+ python3.11 -m venv /home/user/.python_venv3.11/yt-dlp
+ . /home/user/.python_venv3.11/yt-dlp/bin/activate
++ deactivate nondestructive
++ '[' -n '' ']'
++ '[' -n '' ']'
++ '[' -n /bin/sh -o -n '' ']'
++ hash -r
++ '[' -n '' ']'
++ unset VIRTUAL_ENV
++ unset VIRTUAL_ENV_PROMPT
++ '[' '!' nondestructive = nondestructive ']'
++ VIRTUAL_ENV=/home/user/.python_venv3.11/yt-dlp
++ export VIRTUAL_ENV
++ _OLD_VIRTUAL_PATH='/usr/bin:/home/user/bin:~/.cargo/bin:~/.local/bin:/usr/lib/ccache/bin/:/home/user/bin:/usr/bin:/home/user/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/16/bin:/usr/lib/llvm/15/bin:/etc/eselect/wine/bin:/usr/lib64/opencascade/bin'
++ PATH='/home/user/.python_venv3.11/yt-dlp/bin:/usr/bin:/home/user/bin:~/.cargo/bin:~/.local/bin:/usr/lib/ccache/bin/:/home/user/bin:/usr/bin:/home/user/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/16/bin:/usr/lib/llvm/15/bin:/etc/eselect/wine/bin:/usr/lib64/opencascade/bin'
++ export PATH
++ '[' -n '' ']'
++ '[' -z '' ']'
++ _OLD_VIRTUAL_PS1=
++ PS1='(yt-dlp) '
++ export PS1
++ VIRTUAL_ENV_PROMPT='(yt-dlp) '
++ export VIRTUAL_ENV_PROMPT
++ '[' -n /bin/sh -o -n '' ']'
++ hash -r
+ pip install git+https://github.com/yt-dlp/yt-dlp
Collecting git+https://github.com/yt-dlp/yt-dlp
  Cloning https://github.com/yt-dlp/yt-dlp to /tmp/pip-req-build-lxrpmiub
  Running command git clone --filter=blob:none --quiet https://github.com/yt-dlp/yt-dlp /tmp/pip-req-build-lxrpmiub
  Resolved https://github.com/yt-dlp/yt-dlp to commit fcd6a76adc49d5cd8783985c7ce35384b72e545f
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Collecting mutagen (from yt-dlp==2023.7.6)
  Downloading mutagen-1.46.0-py3-none-any.whl (193 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 193.6/193.6 kB 419.1 kB/s eta 0:00:00
Collecting pycryptodomex (from yt-dlp==2023.7.6)
  Obtaining dependency information for pycryptodomex from https://files.pythonhosted.org/packages/45/09/ce11344724e674957402b5f3c0d663388f755ecd1c864f0c405b5f959bd8/pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Downloading pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.4 kB)
Collecting websockets (from yt-dlp==2023.7.6)
  Downloading websockets-11.0.3-cp311-cp311-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (130 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 130.6/130.6 kB 2.1 MB/s eta 0:00:00
Collecting certifi (from yt-dlp==2023.7.6)
  Obtaining dependency information for certifi from https://files.pythonhosted.org/packages/4c/dd/2234eab22353ffc7d94e8d13177aaa050113286e93e7b40eae01fbf7c3d9/certifi-2023.7.22-py3-none-any.whl.metadata
  Downloading certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)
Collecting brotli (from yt-dlp==2023.7.6)
  Downloading Brotli-1.0.9-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.7/2.7 MB 1.3 MB/s eta 0:00:00
Downloading certifi-2023.7.22-py3-none-any.whl (158 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 158.3/158.3 kB 2.9 MB/s eta 0:00:00
Downloading pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 1.9 MB/s eta 0:00:00
Building wheels for collected packages: yt-dlp
  Building wheel for yt-dlp (pyproject.toml) ... done
  Created wheel for yt-dlp: filename=yt_dlp-2023.7.6-py2.py3-none-any.whl size=2799662 sha256=dc533d5db961cf793ab2cc4f3051c28d7d0ef07fa0701da71aa0dd2ab69ffe07
  Stored in directory: /tmp/pip-ephem-wheel-cache-dezi1zm_/wheels/0b/75/e2/28dc66d3b9276340552f0951ba08c6c17f0ec3811090b2bf0c
Successfully built yt-dlp
Installing collected packages: brotli, websockets, pycryptodomex, mutagen, certifi, yt-dlp
Successfully installed brotli-1.0.9 certifi-2023.7.22 mutagen-1.46.0 pycryptodomex-3.18.0 websockets-11.0.3 yt-dlp-2023.7.6
+ echo '. /home/user/.python_venv3.11/yt-dlp/bin/activate'
. /home/user/.python_venv3.11/yt-dlp/bin/activate

$ . /home/user/.python_venv3.11/yt-dlp/bin/activate
(yt-dlp) 
$ yt-dlp -vU https://twitter.com/playstrumpcard/status/1695424220702888009
[debug] Command-line config: ['-vU', 'https://twitter.com/playstrumpcard/status/1695424220702888009']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481]
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.5 (CPython x86_64 64bit) - Linux-6.4.0-gentoo-x86_64-x86_64-Intel-R-_Core-TM-_i7-4910MQ_CPU_@_2.90GHz-with-glibc2.38 (OpenSSL 3.1.2 1 Aug 2023, glibc 2.38)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.07.22, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1864 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[twitter] Extracting URL: https://twitter.com/playstrumpcard/status/1695424220702888009
[twitter] 1695424220702888009: Downloading guest token
[twitter] 1695424220702888009: Downloading GraphQL JSON
ERROR: [twitter] 1695424220702888009: No video could be found in this tweet
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/twitter.py", line 1410, in _real_extract
    self.raise_no_formats('No video could be found in this tweet', expected=True)
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 1193, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)

(yt-dlp)
```

**Comment by Geremia** [comment]
`master` only downloads the video for me (no audio):

for example:
```
$ yt-dlp --verbose https://twitter.com/TuckerCarlson/status/1694513603251241143
[debug] Command-line config: ['--verbose', 'https://twitter.com/TuckerCarlson/status/1694513603251241143']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481]
[debug] Lazy loading extractors is disabled
[debug] Python 3.9.17 (CPython x86_64 64bit) - Linux-6.1.42-x86_64-AMD_Ryzen_Threadripper_2990WX_32-Core_Processor-with-glibc2.37 (OpenSSL 3.1.2 1 Aug 2023, glibc 2.37)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2020.12.05, mutagen-1.46.0, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1864 extractors
[twitter] Extracting URL: https://twitter.com/TuckerCarlson/status/1694513603251241143
[twitter] 1694513603251241143: Downloading guest token
[twitter] 1694513603251241143: Downloading GraphQL JSON
[debug] [twitter] Extracting from video info: 1694464943070822400
[twitter] 1694513603251241143: Downloading m3u8 information
[debug] Sort order given by extractor: res, br, size, proto
[debug] Formats sorted by: hasvid, ie_pref, res, br, size, proto, lang, quality, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, asr, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 1694464943070822400: Downloading 1 format(s): http-2176
[debug] Invoking http downloader on "https://video.twimg.com/amplify_video/1694464943070822400/vid/1280x720/92wPE8yr8DsnkSay.mp4?tag=14"
[download] Resuming download at byte 62913536
[download] Destination: Tucker Carlson - Ep. 19  Debate Night with Donald J Trump [1694464943070822400].mp4
[download] 100% of  159.21MiB in 00:00:07 at 21.41MiB/s
```

**Comment by jakeogh** [comment]
@Geremia I can not reproduce that on git-latest. I get both the audio and video for 1694513603251241143. Can you please test with the venv setup I gave an example of above?

**Comment by bashonly** [comment]
@jakeogh
`pip install git+https://github.com/yt-dlp/yt-dlp` does not install master
use this:
```
python3 -m pip install -U pip setuptools wheel
python3 -m pip install --force-reinstall https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz
```
https://github.com/yt-dlp/yt-dlp/wiki/Installation#pip-master

@Geremia
it seems to be a problem with the source file. Adding `-S proto:m3u8` to your command will get you a format with working audio (same video resolution). I can't find another twitter video that has this problem

**Comment by Geremia** [comment]
@jakeogh I get the same issue in a `venv`. It's strange, because when I open up the mp4 in VLC, it shows an audio track, but VLC keeps it muted.
`yt-dlp -x` does get an m4a of the audio, though.
It seems it has something to do with how `yt-dlp` multiplexes it; `mkvtoolnix-gui` is able to created a properly multiplexed MKV of the MP4. Maybe my issue is unrelated to this Twitter issue.

**Comment by bashonly** [comment]
@Geremia it is an https format, yt-dlp just downloads it byte-for-byte to disk, it does not mux at all

**Comment by Geremia** [comment]
@bashonly I thought "`Default format spec: bestvideo*+bestaudio/best`" sometimes multiplexes, depending on the available formats.

**Comment by bashonly** [comment]
@Geremia it does sometimes, when there are separate video and audio formats that need to be merged. Twitter serves pre-merged video+audio formats, so no muxing takes place with its https formats.

I can reproduce the issue with mpv, which is why I think it is a problem with the source file (twitter serves the m3u8 stream in browser, not the direct https mp4)

**Comment by jakeogh** [comment]
@bashonly 

Thanks, it's not well known, but that pip syntax in my original example (git+https://) installs whatever the latest HEAD is, which is why I noted the commit. Below is the example modified to use the official "install master" commans, the problem remains.

```
$ cat ~/_myapps/yt-dlp/install_venv.sh 
#!/bin/sh

set -o nounset
set -x

venv="$HOME/.python_venv3.11/yt-dlp"

mkdir -p "${venv}"
python3.11 -m venv "${venv}"
. "${venv}/bin/activate"
#pip install --force-reinstall git+https://github.com/yt-dlp/yt-dlp
#pip install git+https://github.com/yt-dlp/yt-dlp

python3 -m pip install -U pip setuptools wheel
python3 -m pip install --force-reinstall https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz

echo ". ${venv}/bin/activate"

$ ~/_myapps/yt-dlp/install_venv.sh 
+ venv=/home/user/.python_venv3.11/yt-dlp
+ mkdir -p /home/user/.python_venv3.11/yt-dlp
+ python3.11 -m venv /home/user/.python_venv3.11/yt-dlp
+ . /home/user/.python_venv3.11/yt-dlp/bin/activate
++ deactivate nondestructive
++ '[' -n '' ']'
++ '[' -n '' ']'
++ '[' -n /bin/sh -o -n '' ']'
++ hash -r
++ '[' -n '' ']'
++ unset VIRTUAL_ENV
++ unset VIRTUAL_ENV_PROMPT
++ '[' '!' nondestructive = nondestructive ']'
++ VIRTUAL_ENV=/home/user/.python_venv3.11/yt-dlp
++ export VIRTUAL_ENV
++ _OLD_VIRTUAL_PATH='/usr/bin:/home/user/bin:~/.cargo/bin:~/.local/bin:/usr/lib/ccache/bin/:/home/user/bin:/usr/bin:/home/user/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/16/bin:/usr/lib/llvm/15/bin:/etc/eselect/wine/bin:/usr/lib64/opencascade/bin'
++ PATH='/home/user/.python_venv3.11/yt-dlp/bin:/usr/bin:/home/user/bin:~/.cargo/bin:~/.local/bin:/usr/lib/ccache/bin/:/home/user/bin:/usr/bin:/home/user/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin:/usr/lib/llvm/16/bin:/usr/lib/llvm/15/bin:/etc/eselect/wine/bin:/usr/lib64/opencascade/bin'
++ export PATH
++ '[' -n '' ']'
++ '[' -z '' ']'
++ _OLD_VIRTUAL_PS1=
++ PS1='(yt-dlp) '
++ export PS1
++ VIRTUAL_ENV_PROMPT='(yt-dlp) '
++ export VIRTUAL_ENV_PROMPT
++ '[' -n /bin/sh -o -n '' ']'
++ hash -r
+ python3 -m pip install -U pip setuptools wheel
Requirement already satisfied: pip in /home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages (23.2.1)
Requirement already satisfied: setuptools in /home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages (68.1.2)
Collecting wheel
  Obtaining dependency information for wheel from https://files.pythonhosted.org/packages/b8/8b/31273bf66016be6ad22bb7345c37ff350276cfd46e389a0c2ac5da9d9073/wheel-0.41.2-py3-none-any.whl.metadata
  Using cached wheel-0.41.2-py3-none-any.whl.metadata (2.2 kB)
Using cached wheel-0.41.2-py3-none-any.whl (64 kB)
Installing collected packages: wheel
Successfully installed wheel-0.41.2
+ python3 -m pip install --force-reinstall https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz
Collecting https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz
  Downloading https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz (2.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5/2.5 MB 2.7 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Collecting mutagen (from yt-dlp==2023.7.6)
  Using cached mutagen-1.46.0-py3-none-any.whl (193 kB)
Collecting pycryptodomex (from yt-dlp==2023.7.6)
  Obtaining dependency information for pycryptodomex from https://files.pythonhosted.org/packages/45/09/ce11344724e674957402b5f3c0d663388f755ecd1c864f0c405b5f959bd8/pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
  Using cached pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.4 kB)
Collecting websockets (from yt-dlp==2023.7.6)
  Using cached websockets-11.0.3-cp311-cp311-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (130 kB)
Collecting certifi (from yt-dlp==2023.7.6)
  Obtaining dependency information for certifi from https://files.pythonhosted.org/packages/4c/dd/2234eab22353ffc7d94e8d13177aaa050113286e93e7b40eae01fbf7c3d9/certifi-2023.7.22-py3-none-any.whl.metadata
  Using cached certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)
Collecting brotli (from yt-dlp==2023.7.6)
  Using cached Brotli-1.0.9-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.7 MB)
Using cached certifi-2023.7.22-py3-none-any.whl (158 kB)
Using cached pycryptodomex-3.18.0-cp35-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.1 MB)
Building wheels for collected packages: yt-dlp
  Building wheel for yt-dlp (pyproject.toml) ... done
  Created wheel for yt-dlp: filename=yt_dlp-2023.7.6-py2.py3-none-any.whl size=2799564 sha256=81c6d4d733d83509965146b216d16acc7d0b3d555f19cc507aa7c109ba7ad1d1
  Stored in directory: /tmp/pip-ephem-wheel-cache-ks8gno6b/wheels/2d/79/97/7209650ef73114e0fe0603480da012ad3afacb9cae6b8acd9a
Successfully built yt-dlp
Installing collected packages: brotli, websockets, pycryptodomex, mutagen, certifi, yt-dlp
Successfully installed brotli-1.0.9 certifi-2023.7.22 mutagen-1.46.0 pycryptodomex-3.18.0 websockets-11.0.3 yt-dlp-2023.7.6
+ echo '. /home/user/.python_venv3.11/yt-dlp/bin/activate'
. /home/user/.python_venv3.11/yt-dlp/bin/activate

$ . /home/user/.python_venv3.11/yt-dlp/bin/activate
(yt-dlp) 
$ yt-dlp -vU https://twitter.com/playstrumpcard/status/1695424220702888009
[debug] Command-line config: ['-vU', 'https://twitter.com/playstrumpcard/status/1695424220702888009']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481]
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.5 (CPython x86_64 64bit) - Linux-6.4.0-gentoo-x86_64-x86_64-Intel-R-_Core-TM-_i7-4910MQ_CPU_@_2.90GHz-with-glibc2.38 (OpenSSL 3.1.2 1 Aug 2023, glibc 2.38)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.07.22, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1864 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[twitter] Extracting URL: https://twitter.com/playstrumpcard/status/1695424220702888009
[twitter] 1695424220702888009: Downloading guest token
[twitter] 1695424220702888009: Downloading GraphQL JSON
ERROR: [twitter] 1695424220702888009: No video could be found in this tweet
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/twitter.py", line 1410, in _real_extract
    self.raise_no_formats('No video could be found in this tweet', expected=True)
  File "/home/user/.python_venv3.11/yt-dlp/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 1193, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)

(yt-dlp) 
$   
```

**Comment by bashonly** [comment]
@jakeogh ah you're right, my bad.

It is a bug in the extractor. I will have a PR up soon
